src/core/constants.js

```diff
@@ -190,17 +190,34 @@ export const UNINSTALL_COMPLETE = 'UNINSTALL_COMPLETE';
 export const INSTALL_ERROR = 'INSTALL_ERROR';
 
 // Tracking install actions.
+export const ENABLE_ACTION = 'enable';
 export const INSTALL_ACTION = 'install';
+export const INSTALL_CANCELLED_ACTION = 'install:cancelled';
+export const INSTALL_ERRORED_ACTION = 'install:errored';
+export const INSTALL_FAILED_ACTION = 'install:failed';
```

Comment:
what's the difference between errored and failed here? is install failed specific to that and error any other kind of error?

Reply:
I'm going to remove those FAILED/ERRORED events for now.

Reply:
so you'll have a follow up issue for this?